nixos/modules/services/web-apps/moin.nix

```diff
@@ -0,0 +1,273 @@
+{ config, lib, pkgs, ... }:
+with lib;
+
+let
+  cfg = config.services.moin;
+  pkg = pkgs.python27Packages.moinmoin;
+  dataDir = "/var/lib/moin";
```

Comment:
@mmilata or you could leave it hardcoded the way you have it now, taking advantage of `systemd` features to manage your state directory and @danieldk could make a symlink. Dealers choice :smile:

Note that currently you are using `tmpfiles.d` to provision directories that `StateDirectory` could be used to provision if you leave the data directory hardcoded to `/var/lib/*`.

Reply:
From the feedback on my previous module PR I figured it's preferred to have `dataDir` hardcoded under `/var/lib`. You can also bind-mount the external storage as `/var/lib/moin`, this way you should be able to configure systemd to wait for the mount before starting the wiki.

@aanderse I'm a fan of the systemd sandboxing options that `StateDirectory` enables, however this module doesn't have it's own unit file as everything is run under uWSGI daemon. Maybe using e.g. `spawn-fcgi` instead of uWSGI would be better in this regard, the wiki supports FastCGI as well, not sure about performance though.

Reply:
@mmilata ah sorry. I just gave the module a quick look over and missed that fact. :+1: